matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/user/accountdata/UserAccountDataDataSource.kt

```diff
@@ -60,6 +60,16 @@ internal class UserAccountDataDataSource @Inject constructor(
         )
     }
 
+    fun getAccountDataEventsStartWith(type: String): List<UserAccountDataEvent> {
+        return realmSessionProvider.withRealm { realm ->
+            realm
+                    .where(UserAccountDataEntity::class.java)
+                    .contains(UserAccountDataEntityFields.TYPE, type)
```

Comment:
I think we should use `beginsWith` instead of `contains` if we want to reflect the method name.